canvas_modules/harness/src/client/components/custom-canvases/parallax/parallax.scss

```diff
@@ -21,6 +21,10 @@
 	// ----------------------------------------------------
 	.d3-node-group {
 
+		.d3-node-label{
+			color: black;
```

Comment:
I thought you were going to us a Carbon token here  - right?

Reply:
I tested all of the `text tokens` but the different shades of gray didn't seem to sit well with some of the lighter nodes.